korman/ui/modifiers/anim.py

```diff
@@ -40,9 +41,13 @@ def animation(modifier, layout, context):
     if action:
         layout.prop_search(modifier, "initial_marker", action, "pose_markers", icon="PMARKER")
         col = layout.column()
-        col.enabled = modifier.loop
+        col.enabled = modifier.loop and not modifier.obj_sdl_anim
         col.prop_search(modifier, "loop_start", action, "pose_markers", icon="PMARKER")
         col.prop_search(modifier, "loop_end", action, "pose_markers", icon="PMARKER")
+        col = layout.column()
```

Comment:
I'm wondering if we should just add in a `layout.separator()` here and make this the full width.

Reply:
This causes the field to vanish. 😐

Reply:
Did you also change over the next two things from `col` to `layout`?

Reply:
Fixed now.